Jenkinsfile

```
@@ -33,6 +34,18 @@ pipeline {
           recordIssues enabledForFailure: true, tool: checkStyle()
           recordIssues enabledForFailure: true, tool: spotBugs()
         }
+        success {
+            stash name: 'binary', includes: 'target/plugin-health-scoring.jar,src/main/docker/Dockerfile'
```

```suggestion
            stash name: 'binary', includes: 'target/plugin-health-scoring.jar'
```
After discussion with @dduportal [I've modified my pipeline library PR](https://github.com/jenkins-infra/pipeline-library/pull/465/commits/652268cdcae3e510fe9f39f6ce1a9ceec11e7e2e) in order to be more generic by doing the `checkout` even if `unstash` isn't empty. (To avoid mistake by other users)
Thus, stashing only the resulting .jar is enough here.